nft-simple/src/mint.rs

```diff
@@ -0,0 +1,27 @@
+use crate::*;
+
+#[near_bindgen]
+impl Contract {
+    #[payable]
+    pub fn nft_mint(&mut self, token_id: TokenId, metadata: String) {
+        let initial_storage_usage = env::storage_usage();
+        self.assert_owner();
+        let token = Token {
+            owner_id: self.owner_id.clone(),
```

Comment:
Feels like the `owner_id` would. be something we'd pass in as a parameter to `nft_mint(…)` to me

Reply:
why?